dockerfiles/rust/1.75/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:22.04
+
+RUN apt-get update
+
+# cargo 1.75.0 is latest version on apt as of 9/7/2024
+RUN apt install cargo=1.75.0+dfsg0ubuntu1~bpo0-0ubuntu0.22.04 -y
```

Comment:
Thanks for specifying the version. However, the preferred way to install Rust is through rustup. There is a couple shell commands you could put together to install through rustup non-iteractively. You can also check out https://github.com/Submitty/DockerImagesRPI/blob/011e7f1981e1639e95a2de846892391d577ada30/dockerfiles/csci4510/spring24/Dockerfile#L273-L307 which is how DSA fetches Rust for its Docker image.

Reply:
I avoided using rustup due to it installing for the current user and not systemwide. I want to avoid any complications surrounding the autograder login, hence apt instead.

Reply:
Changes made and tested